<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QWidget" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MENU</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(134, 189, 255);
border-color: rgb(13, 13, 13);</string>
  </property>
  <widget class="QPushButton" name="pbIngreso">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #333;
border: 1px solid #555;
border-radius: 9px;
padding: 5px;
background-color: rgb(222, 222, 222);</string>
   </property>
   <property name="text">
    <string>INGRESAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSalida">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #333;
border: 1px solid #555;
border-radius: 9px;
padding: 5px;
background-color: rgb(222, 222, 222);</string>
   </property>
   <property name="text">
    <string>SALIDA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAutos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #333;
border: 1px solid #555;
border-radius: 9px;
padding: 5px;
background-color: rgb(222, 222, 222);</string>
   </property>
   <property name="text">
    <string>AUTOS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCerrarSesion">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #333;

padding: 5px;
background-color: rgb(222, 222, 222);</string>
   </property>
   <property name="text">
    <string>CERRAR SESION</string>
   </property>
  </widget>
  <widget class="QLabel" name="lIcono2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recursos.qrc">:/imagenes/imagenes/auto.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
